Slider/Slider.jsx

```diff
@@ -13,27 +13,36 @@ export default class Slider extends MaterialComponent {
     this._onChange = this._onChange.bind(this);
     this._onInput = this._onInput.bind(this);
   }
-  _onChange() {
+
+  _onChange(e) {
     if (this.props.onChange) {
-      this.props.onChange(this.MDComponent.value);
+      this.props.onChange(e);
```

Comment:
can you please confirm that some value is passed here?
last time i check nothing was passed hence, this was passed

Reply:
The MDCComponent `listen` method attaches an event listener on the DOM element (see [packages/mdc-base/component.js#L89](https://github.com/material-components/material-components-web/blob/master/packages/mdc-base/component.js#L89)).

In the image below is an example of the output of both `onInput` and `onChange`. This is the code that outputs to the console: `console.log(evt, slider.getValue());`.
![image](https://user-images.githubusercontent.com/2348303/34499234-41717364-f005-11e7-8da1-930d60b59efb.png)